x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluation_quality_metrics_table.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { EuiAccordion, EuiInMemoryTable, EuiPanel } from '@elastic/eui';
+
+import { MetricItem } from '../../../../common/analytics';
+
+const columns = [
+  {
+    field: 'className',
+    name: i18n.translate(
+      'xpack.ml.dataframe.analytics.classificationExploration.recallAndAccuracyClassColumn',
+      {
+        defaultMessage: 'Class',
+      }
+    ),
+    sortable: true,
+    truncateText: true,
+  },
+  {
```

Comment:
Nit - if the Accuracy and Recall columns were switched round, then the order would match that for the overall stats above:

![image](https://user-images.githubusercontent.com/7405507/128743959-2fffb574-8fc0-4f6c-bc74-a51e512f7156.png)